Course_Part_03/src/Assignment11_StringArray.java

```diff
@@ -0,0 +1,11 @@
+public class Assignment11_StringArray {
+
+    public static void main(String[] args) {
+
+        String[] daysArray = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"};
```

Comment:
de array zou ik `daysOfTheWeek` noemen, het is niet zo interessant om aan te geven in de naam of het een array betreft.

Verder prima uitgewerkt!